highwayhash/vector256.h

```diff
@@ -287,7 +287,12 @@ class V256<uint64_t> {
 
   // Broadcasts i to all lanes.
   HH_INLINE explicit V256(T i)
+#if HH_ARCH_X86
+      // _mm_cvtsi64_si128 is not available on x86
+      : V256(i,i,i,i) {}
```

Comment:
This likely won't work on MSVC. How about we use _mm256_set_epi32 instead?

Reply:
This works on MSVC toolkit 170 but I've changed it to use _mm_insert_epi32, which should generate better code I think.